### PR TITLE
Legacy compatible

### PR DESCRIPTION
**Pull request by parar020100**
All drmatt/tardis/* files and folders renamed into drmatt/legtar/* ones
+ changed spawnmenu category from "Doctor Who" to "Doctor Who - TARDIS (Legacy)" 
+ changed name to Legacy TARDIS

### CODE REVIEW

File: addon.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-	"title"		:	"TARDIS",
+	"title"		:	"Legacy TARDIS",
```
Comment (MattJeanes) [inline addon.json:2]: Change this to TARDIS Legacy

Comment (MattJeanes) [reply addon.json:2]:
```suggestion
	"title"		:	"TARDIS Legacy",
```

File: lua/autorun/tardis_options.lua [hunk 2]
```diff
@@ -54,7 +54,7 @@ CreateClientConVar("tardis_extcol_g", "255", true, true)
 CreateClientConVar("tardis_extcol_b", "255", true, true)
 
 hook.Add("PopulateToolMenu", "TARDIS-PopulateToolMenu", function()
-	spawnmenu.AddToolMenuOption("Options", "Doctor Who", "TARDIS_Options", "TARDIS", "", "", function(panel)
+	spawnmenu.AddToolMenuOption("Options", "Doctor Who", "TARDIS_Options", "Legacy TARDIS", "", "", function(panel)
```
Comment (MattJeanes) [inline lua/autorun/tardis_options.lua:57]:
```suggestion
	spawnmenu.AddToolMenuOption("Options", "Doctor Who", "TARDIS_Options", "TARDIS Legacy", "", "", function(panel)
```

File: lua/entities/sent_tardis/shared.lua [hunk 3]
```diff
@@ -4,15 +4,15 @@ if WireLib then
 else
 	ENT.Base			= "base_gmodentity"
 end 
-ENT.PrintName		= "TARDIS"
+ENT.PrintName		= "Legacy TARDIS"
```
Comment (MattJeanes) [inline lua/entities/sent_tardis/shared.lua:7]:
```suggestion
ENT.PrintName		= "TARDIS Legacy"
```

File: lua/entities/sent_tardis/shared.lua [hunk 4]
```diff
@@ -4,15 +4,15 @@ if WireLib then
...
 ENT.Author			= "Dr. Matt"
 ENT.Contact			= "mattjeanes23@gmail.com"
 ENT.Instructions	= "Use with the sonic or press E to pilot."
 ENT.Purpose			= "Time and Relative Dimension in Space"
 ENT.Spawnable		= true
 ENT.AdminSpawnable	= true
 ENT.RenderGroup = RENDERGROUP_BOTH
-ENT.Category		= "Doctor Who"
+ENT.Category		= "Doctor Who - TARDIS (Legacy)"
```
Comment (MattJeanes) [inline lua/entities/sent_tardis/shared.lua:15]: This should not be changed
```suggestion
ENT.Category		= "Doctor Who"
```

**Review by MattJeanes — CHANGES_REQUESTED** [review]
Model/material/sound changes look good but I disagree with the naming changes, can we please correct these?

File: lua/entities/sent_tardis/shared.lua (hunk 3 above)
Comment (parar020100) [inline lua/entities/sent_tardis/shared.lua:7]:
```suggestion
ENT.PrintName		= " TARDIS Legacy "
```

File: lua/entities/sent_tardis/shared.lua (hunk 4 above)
Comment (parar020100) [inline lua/entities/sent_tardis/shared.lua:15]:
```suggestion
ENT.Category		= "Doctor Who - TARDIS"
```

**Review by MattJeanes — APPROVED** [review]

**Pull request merged**